playback/src/player.rs

```diff
@@ -1330,6 +1311,47 @@ impl PlayerInternal {
         self.limiter_strength = 0.0;
     }
 
+    fn get_normalisation_factor(config: &PlayerConfig, data: NormalisationData) -> f64 {
```

Comment:
I'll move this back to `NormalisationData::get_factor`. Won't matter for the behavior, but this move wasn't necessary.

Reply:
Done in 5b63ef6.